frontend-project/src/pages/administrativeUnits/list/locales/pl-Pl.ts

```diff
@@ -0,0 +1,8 @@
+export default {
+  'administrative-units-list.page-header-content': 'Lista jednostek administracyjnych',
+  'administrative-units-list.table.columns.name.title': 'Nazwa',
+  'administrative-units-list.table.columns.category.title': 'Kategoria',
+  'administrative-units-list.table.columns.slug.title': 'Oznaczenie', // ? nie tłumaczyłem nigdy slug na polski
```

Comment:
W Watchdogu stosujemy na to `Identyfikator redakcyjny` w pozostałych projektach.